src/cls/rbd/cls_rbd.cc

```diff
@@ -711,6 +717,98 @@ int set_stripe_unit_count(cls_method_context_t hctx, bufferlist *in, bufferlist
   return 0;
 }
 
+/**
+ * get the image flags
+ *
+ * Input:
+ * @param snap_id which snapshot to query, to CEPH_NOSNAP (uint64_t)
```

Comment:
The object map could have been valid historically and invalid at the current state of the image.  I was planning on having the flags support more than just the object map validity bit in the future (i.e. someone recently suggested a way to disable caching on a per-image basis).